SimGeneral/MixingModule/python/caloTruthProducer_cfi.py

```diff
@@ -40,6 +40,12 @@
     simHitCollections = dict(hgc = {2 : cms.InputTag('g4SimHits','HGCHitsHEback')} ),
 )
 
+from Configuration.Eras.Modifier_phase2_hfnose_cff import phase2_hfnose
+phase2_hfnose.toModify(
+    caloParticles,
+    simHitCollections = dict(hgc = {2 : cms.InputTag('g4SimHits','HFNoseHits')} ),
```

Comment:
Just to make sure, the intention is to replace `g4SimHits:HGCHitsHEback` with `g4SimHits:HFNoseHits`?

@kpedro88

Reply:
@makortel thanks for pointing this out, it is not correct. HFnose hits should be appended, so 2 -> 3

Reply:
Just changing `2` to `3` won't work because the original `VInputTag` has only 3 elements. To append one needs e.g. something like
```python
simHitCollections = dict(hgc = caloPartices.simHitCollections.hgc + [cms.InputTag('g4SimHits','HFNoseHits')])
```

Reply:
@makortel thanks for the suggestion